pkg/helm/charts/tempo/tempo.go

```diff
@@ -0,0 +1,200 @@
+// Package tempo provides a Helm chart for installing:
+// - https://github.com/grafana/helm-charts/tree/main/charts/tempo
+// - https://grafana.com/oss/tempo/
+package tempo
+
+import (
+	"bytes"
+	"text/template"
+
+	"github.com/oslokommune/okctl/pkg/config"
+
+	"github.com/oslokommune/okctl/pkg/helm"
+)
+
+// New returns an initialised Helm chart for installing cluster-tempo
+func New(values *Values) *helm.Chart {
+	return &helm.Chart{
+		RepositoryName: "grafana",
+		RepositoryURL:  "https://grafana.github.io/helm-charts",
+		ReleaseName:    "tempo",
+		Version:        "0.6.3",
+		Chart:          "tempo",
+		Namespace:      "monitoring",
+		Timeout:        config.DefaultChartApplyTimeout,
+		Values:         values,
+	}
+}
+
+// NewDefaultValues returns the mapped values.yml containing
+// the default values
+func NewDefaultValues() *Values {
+	return &Values{}
+}
+
+// Values contains the required inputs for generating the values.yml
+// One of those cases where there really isn't much to change, but
+// I will leave these structures here nonetheless.
+// We should configuring and setting up a S3 storage backend:
+// - https://github.com/grafana/tempo/blob/master/docs/tempo/website/configuration/s3.md
+type Values struct{}
+
+// RawYAML implements the raw marshaller interface in the Helm package
+func (v *Values) RawYAML() ([]byte, error) {
+	tmpl, err := template.New("values").Parse(valuesTemplate)
+	if err != nil {
+		return nil, err
+	}
+
+	var buff bytes.Buffer
+
+	err = tmpl.Execute(&buff, *v)
+	if err != nil {
+		return nil, err
+	}
+
+	return buff.Bytes(), nil
+}
+
+// nolint: lll
```

Comment:
Should we put a comment with an URL to the source of this values.yaml? Makes it a bit simpler when upgradring in the future (because values.yaml is tied to the tempo helm chart version), and if we for some other reason want to compare.

Reply:
Yes! Very good idea ❤️